### PR TITLE
AT_01.03.027 | Header, User icon opens a dropdown menu and redirects to the selected menu-list option

### PR DESCRIPTION
**Pull request by LenaAndreeva**
https://trello.com/c/UMr88AKW/1394-at0103027-header-user-icon-is-visible-clickable-opens-a-dropdown-menu-and-redirects-to-the-selected-menu-list-option

### CODE REVIEW

**Review by lenashlyakhovets — APPROVED** [review]

File: cypress/e2e/userIcon_zb.cy.js [hunk 1]
```diff
@@ -50,6 +50,14 @@ describe('Header | User icon', () => {
         cy.get('.first-of-type span').contains('Credentials').click();
         cy.get('.jenkins-app-bar__content h1').should('have.text', 'Credentials');
     })
+
+    it('AT_01.03.027 | <Header> User icon is visible, clickable, opens a dropdown menu and redirects', () => {
+        cy.get('a[href="/user/admin"]').should('be.visible');
+        cy.get('button:nth-child(3)').click({force: true});
+        cy.get('#breadcrumb-menu').should('be.visible');
+        cy.get('a[href="/user/admin/builds"').click();
+        cy.get('#main-panel h1').should('have.text', 'Builds for admin');
```
Comment (KatyaKaterina33) [inline cypress/e2e/userIcon_zb.cy.js:59]: It's better to move 'Builds for admin' text in fixures.

**Comment by KatyaKaterina33** [comment]
Please check PR link in AT card.

**Comment by S-Lanchik** [comment]
Your test shouldn't have many checks. Best practice: one test - one check.

File: cypress/e2e/userIcon_zb.cy.js (hunk 1 above)
Comment (LenaAndreeva) [reply cypress/e2e/userIcon_zb.cy.js]: Thanks for your review! I moved the text to the fixtures and fixed PR link in my AT card.

**Comment by LenaAndreeva** [comment]
> > Your test shouldn't have many checks. Best practice: one test - one check.

Thank you for your advice. I left only one check.

**Review by ozhegov — APPROVED** [review]

**Comment by KatyaKaterina33** [comment]
The test failed with my user name, it would be better to replace user "admin" with a user name running the test, in this case to avoid errors it is recommended to use the toLowerCase() function: 
const username = Cypress.env('local.admin.username').toLowerCase();

**Comment by LenaAndreeva** [comment]
> 

I should write this function directly in my test, right?

**Comment by KatyaKaterina33** [comment]
I think the "userName" variable can be defined in the spec file itself, before your test, also it's necessary to change the "admin" text in the test for this variable.

**Pull request merged**